odk1-src/form-widgets.rst

```diff
@@ -1725,8 +1725,8 @@ and are not visible to the user.
 These items are dependent on the survey collection event:
 
 - :tc:`start` --- The time the survey was started.
-- :tc:`end` --- The time the survey was completed.
-- :tc:`today` --- The date of survey collection.
+- :tc:`end` --- The last time the survey was saved.
+- :tc:`today` --- The current date, according to the device, when the survey is collected. (The date is determined when the widget is rendered.)
```

Comment:
Ugh, now that I look at this, the meta widget isn't shown. Let's just drop the stuff in the parenthesis. Better to say nothing than say wrong stuff.